proton-j/src/main/java/org/apache/qpid/proton/engine/impl/ssl/DefaultSslEngineFacade.java

```diff
@@ -36,7 +36,7 @@
     /**
      * Our testing has shown that application buffers need to be a bit larger
      * than that provided by {@link SSLSession#getApplicationBufferSize()} otherwise
-     * {@link Status#BUFFER_OVERFLOW} will result on {@link SSLEngine#unwrap()}.
+     * {@link Status#BUFFER_OVERFLOW} will result on {@link SSLEngine#unwrap)}.
```

Comment:
Seems to me like this change breaks the javadoc rather than fixing anything.

Reply:
typo on my part. I meant to delete both of the parens from the `()`, that was my intention (because there is not a parameterless overload of the function)